twindb_backup/destination/s3.py

```diff
@@ -37,21 +45,43 @@
 
 
 class S3Error(DestinationError):
+    """S3 destination errors"""
     pass
 
 
+class AWSAuthOptions(object):  # pylint: disable=too-few-public-methods
```

Comment:
What is the benefit of this separate class? I think this indirection has no benefit.